index.html

```diff
@@ -969,6 +978,50 @@ <h2>
       <p class="issue">TODO add COSE Sign1 detached payload examples</p>
 
     </section>
+    <section>
+      <h3>Selective Disclosure</h3>
```

Comment:
I found this title rather confusing.

Perhaps we could rename it? 

My understanding is that it's a VC that is derived from an SD-JWT. 

```suggestion
      <h3>Credential derived from SD-JWT</h3>
```

Reply:
@Sakurann @bc-pi since I pulled the example from your draft, can you comment on the title of the section that references your example?

Reply:
@andresuribe87 I've added additional context, based on the other PR feedback we received, perhaps its clearer now? in the case that you feel it is not, is your suggestion here blocking?